### PR TITLE
Adding postgresql repl pwd to ory hydra credentials secret

### PR DESCRIPTION
**Pull request by hamdymoussa**
Update secret-migration.sh 

If (Values.postgresql.replication.enabled) is true then ory-hydra-credential secret will have a PostgreSQL replication pwd other wise ory-hydra-credentials won't have a replication pwd.

**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Comment by hamdymoussa** [comment]
Linked with https://github.tools.sap/kyma/backlog/issues/1308

**Comment by mjakobczyk** [comment]
Thank you for your contribution! We are going to change the script anyways in scope of the following task, however: https://github.com/kyma-project/kyma/issues/11629. Closing.

**Comment by strekm** [comment]
as it has high prio and it can't wait for #11629 i will reopen it

File: resources/ory/files/secret-migration.sh [hunk 1]
```diff
@@ -74,36 +80,38 @@ if [[ -z "${COOKIE}" ]]; then
...
 fi
 
+{{- if contains .Values.postgresql.replication.enabled "true" }}
 DATA=$(cat << EOF
   ${DSN_KEY}: $(echo -n "${DSN}" | base64 -w 0)
   ${SECRET_SYSTEM_KEY}: $(echo -n "${SYSTEM}" | base64 -w 0)
   ${SECRET_COOKIE_KEY}: $(echo -n "${COOKIE}" | base64 -w 0)
   {{- if .Values.global.ory.hydra.persistence.enabled }}
   ${PASSWORD_KEY}: $(echo -n "${PASSWORD}" | base64 -w 0)
+  ${PASSWORD_R_KEY}: $(echo -n "${PASSWORDR}" | base64 -w 0)
```
Comment (strekm) [inline resources/ory/files/secret-migration.sh:90]: i think `if else` can be wrapped only around this line or am i missing something?

File: resources/ory/files/secret-migration.sh [hunk 2]
```diff
@@ -74,36 +80,38 @@ if [[ -z "${COOKIE}" ]]; then
...
+  ${SERVICE_ACCOUNT_KEY}: $(echo -n "${SERVICE_ACCOUNT}")
+  {{- end }}
+EOF
+)
+{{- end }}
 
 echo "Data to be applied:"
 echo "${DATA}"
 
-SECRET=$(cat <<EOF
```
Comment (strekm) [inline resources/ory/files/secret-migration.sh:93]: why is this removed?

File: resources/ory/files/secret-migration.sh (hunk 1 above)
Comment (hamdymoussa) [reply resources/ory/files/secret-migration.sh:90]: yup its better thank you for your comment, I'll update.

File: resources/ory/files/secret-migration.sh (hunk 2 above)
Comment (hamdymoussa) [reply resources/ory/files/secret-migration.sh:93]: yes should be kept. it was my error . thanks.

**Comment by hamdymoussa** [comment]
I'll close this PR and i'll open another PR which consider all @strekm comments. thanks @strekm for your comments.

**Comment by hamdymoussa** [comment]
I'll close this PR and i'll open another PR which consider all @strekm comments. thanks @strekm for your comments.
